data/emulator.js

```diff
@@ -1,5 +1,5 @@
 class EmulatorJS {
-    version = 11; //Increase by 1 when cores are updated
+    version = 12; //Increase by 1 when cores are updated
```

Comment:
We removed the cores but it is probably good to increase the number for old versions.